cfme/fixtures/v2v.py

```diff
@@ -19,19 +23,38 @@ def v2v_providers(request, second_provider, provider):
 
 @pytest.fixture(scope='function')
 def host_creds(request, v2v_providers):
-    """Add credentials to conversation host"""
-    provider = v2v_providers[1]
-    hosts = provider.hosts.all()
+    """Add credentials to VMware and RHV hosts."""
+    if len(v2v_providers) > 2:
+        pytest.skip("There are more than two providers in v2v_providers fixture,"
+                    "which is invalid, skipping.")
     try:
-        hosts = hosts if getattr(request, 'param', '') == 'multi-host' else hosts[0:1]
-        for host in hosts:
-            host_data, = [data for data in provider.data['hosts'] if data['name'] == host.name]
+        vmware_provider, rhv_provider = None, None
+        for provider in v2v_providers:
+            if provider.one_of(VMwareProvider):
+                vmware_provider = provider
+            elif provider.one_of(RHEVMProvider):
+                rhv_provider = provider
+        if not vmware_provider and not rhv_provider:
+            pytest.skip("Something Wrong with vmware_provider or rhv_provider, skipping.")
+
+        vmware_hosts = vmware_provider.hosts.all()
+        for host in vmware_hosts:
+            host_data, = [data for data in vmware_provider.data['hosts']
+                        if data['name'] == host.name]
```

Comment:
this indentation looks off, it should align with the 'd' in 'data'

Reply:
which data? there are many data in there :P LOL. and my Flake8Lint does not complain

Reply:
the first data :) pycharm complains it is "under-indented` when I checked out your branch